modules/virt-configuring-interface-link-state-web.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-setting-interface-link-state.adoc
+
+:_mod-docs-content-type: PROCEDURE                                  
+[id="virt-configuring-interface-link-state-web_{context}"]
+= Setting the VM interface link state by using the web console
+
+You can set the link state of a primary or secondary virtual machine (VM) network interface by using the web console.
+
+.Prerequisites
+* You are logged into the {product-title} web console.
+
+.Procedure
+. Navigate to *Virtualization* -> *VirtualMachines*.
+
+. Select a VM to view the *VirtualMachine details* page.
+
+. On the *Configuration* tab, click *Network*. A list of network interfaces is displayed.
+
+. Click the Options menu {kebab} of the interface that you want to edit.
+
+. Choose the appropriate option to set the interface link state:
+.. If the current interface link state is `up`, select *Set link down*.
```

Comment:
```suggestion
** If the current interface link state is `up`, select *Set link down*.
```
Because the user is supposed to complete either substep a or substep b, consider using an unordered list here.